src/TypeCatalogGen/powershell_win7-x64.inc

```diff
@@ -0,0 +1,165 @@
+C:\Users\MarkLocal\AppData\Local\Microsoft\dotnet\sdk\NuGetFallbackFolder\microsoft.applicationinsights\2.4.0\lib\netstandard1.3\Microsoft.ApplicationInsights.dll;
```

Comment:
Please remove the change - the file is auto generated.

Reply:
can we get this file added to the ignore? It is super annoying to need to watch out for this file in my commits all the time.

Reply:
https://github.com/PowerShell/PowerShell/blob/master/src/TypeCatalogGen/.gitignore

Reply:
fixed